game/scripts/npc/abilities/shadow_demon_disruption.txt

```diff
@@ -52,22 +52,22 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "illusion_outgoing_damage"                        "-70.0 -55.0 -40.0 -25.0 -10.0 5.0"
+        "illusion_outgoing_damage"                        "-70.0 -50.0 -30.0 -10.0 10.0 30.0" //OAA
       }
       "04"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "illusion_outgoing_tooltip"                       "30.0 45.0 60.0 75.0 90.0 105.0"
+        "illusion_outgoing_tooltip"                       "30.0 50.0 70.0 90.0 110.0 130.0" //OAA
       }
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "illusion_incoming_damage"                        "200 175 150 125 75 25" //OAA
+        "illusion_incoming_damage"                        "200 200 200 200 100 50" //OAA
```

Comment:
Revert this because it's not ''redundant'. Most SD players max Disruption last unless they are memeing with ''illusion build''. With the scaling (200 175 150 125 75 25) illusions are not dying that fast - they are still bad because of all the nukes that exist but whatever. If you nerf this scaling, you will make the change above (illusion dmg output buff) meaningless and the whole ability would be actually nerfed.